bintray/conan_center_guide.rst

```diff
@@ -0,0 +1,88 @@
+.. _conan_center_flow:
+
+Contributing packages to conan-center
+=====================================
+
+As a moderated and curated repository, `conan-center`_ will not be populated automatically. Initially, it will be empty.
+To have your recipe or binary package available on `conan-center`_, you need to submit an inclusion request to Bintray,
+and the Bintray team will review your request.
+
+- If you are the author of an open source library, your package will be approved.
+  Keep in mind that it is your responsibility to maintain acceptable standards of quality for all packages you submit
+  for inclusion in `conan-center`_.
+
+- If you are packaging a third-party library you need to follow the guidelines below.
+
+
+Inclusion guidelines for third party libraries
+**********************************************
+
+In the inclusion request process, the JFrog staff will perform a general review and will make suggestions for improvements or
+better/cleaner ways to do implement the code.
+
```

Comment:
Can't be other conan package for the same library, even for different versions. Contact the author and collaborate.